app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SessionsController < ActionController::Base
+  def destroy
+    session.clear
+    redirect_to '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.